.github/workflows/release.yml

```
@@ -0,0 +1,26 @@
+name: Release
+on:
+  push:
+    branches: [master, main]
+    tags: ["*"]
+jobs:
+  release:
+    runs-on: ubuntu-18.04
```

```suggestion
    runs-on: ubuntu-20.04
```
same reason as before